app/controllers/RiderStageConnectionController.java

```diff
@@ -48,15 +48,14 @@ public RiderStageConnectionController(StageRepository stageRepository, RiderRepo
         });
     }
 
-
-    public CompletionStage<Result> getStage(int stageId) {
-        return stageRepository.getStage(stageId).thenApplyAsync(stage -> {
-           return ok(stage);
+    public CompletionStage<Result> getRiderStageConnection(long stageId, long riderId) {
+        return riderStageConnectionRepository.getRiderStageConnectionByRiderAndStage(stageId, riderId).thenApplyAsync(riderStageConnection -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)